src/widgets/allergies/allergy-form.component.tsx

```diff
@@ -486,7 +477,7 @@ export default function AllergyForm(props: AllergyFormProps) {
     );
   }
 
-  function editForm() {
+  function editAllergy() {
```

Comment:
Sometime soon, not in this PR, can we split the Edit and View forms out into their own components? Assuming they wouldn't have to have a million parameters.